modules/examples/src/main/java/com/ibm/cloud/platform_services/posture_management/v1/PostureManagementExamples.java

```diff
@@ -0,0 +1,119 @@
+ */
+
+package com.ibm.cloud.platform_services.posture_management.v1;
+
+import com.ibm.cloud.platform_services.posture_management.v1.model.CreateValidationScanOptions;
+import com.ibm.cloud.platform_services.posture_management.v1.model.ListProfileOptions;
+import com.ibm.cloud.platform_services.posture_management.v1.model.ListScopesOptions;
+import com.ibm.cloud.platform_services.posture_management.v1.model.ProfilesList;
+import com.ibm.cloud.platform_services.posture_management.v1.model.Result;
+import com.ibm.cloud.platform_services.posture_management.v1.model.ScopesList;
+import com.ibm.cloud.sdk.core.http.Response;
+import com.ibm.cloud.sdk.core.service.exception.ServiceResponseException;
+import com.ibm.cloud.sdk.core.util.CredentialUtils;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+//
+// This file provides an example of how to use the Posture Management service.
+//
+// The following configuration properties are assumed to be defined:
+// POSTURE_MANAGEMENT_URL=<service base url>
+// POSTURE_MANAGEMENT_AUTH_TYPE=iam
+// POSTURE_MANAGEMENT_APIKEY=<IAM Api key>
+// POSTURE_MANAGEMENT_AUTH_URL=<IAM token service base URL - omit this if using the production environment>
+// POSTURE_MANAGEMENT_ACCOUNT_ID=<IBM Cloud Account ID>
+// POSTURE_MANAGEMENT_PROFILE_NAME=<Name of the Profile>
+// POSTURE_MANAGEMENT_SCOPES_NAME=<Name of the Scope>
+//
+// These configuration properties can be exported as environment variables, or stored
+// in a configuration file and then:
+// export IBM_CREDENTIALS_FILE=<name of configuration file>
+//
+public class PostureManagementExamples {
+  private static final Logger logger = LoggerFactory.getLogger(PostureManagementExamples.class);
+  protected PostureManagementExamples() { }
+
+  @SuppressWarnings("checkstyle:methodlength")
+  public static void main(String[] args) throws Exception {
+    PostureManagement service = PostureManagement.newInstance();
+
+    // Load up our test-specific config properties.
+    Map<String, String> config = CredentialUtils.getServiceProperties(PostureManagement.DEFAULT_SERVICE_NAME);
+
+    String accountId = config.get("ACCOUNT_ID");
+    String profileName = config.get("PROFILE_NAME");
+    String scopesName = config.get("SCOPES_NAME");
+
+    Long profileId = null;
+    Long scopeId = null;
+
+    try {
+      // begin-list_profile
+      ListProfileOptions listProfileOptions = new ListProfileOptions.Builder()
+        .accountId(accountId)
+        .name(profileName)
+        .build();
+
+      Response<ProfilesList> response = service.listProfile(listProfileOptions).execute();
+      ProfilesList profilesList = response.getResult();
+
+      System.out.println(profilesList);
+      // end-list_profile
+      profileId = profilesList.getProfiles().get(0).getProfileId();
+    } catch (ServiceResponseException e) {
+        logger.error(String.format("Service returned status code %s: %s\nError details: %s",
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.

Reply:
@rubieHess thanks for your suggestion. It will be fixed soon. Other examples are going to be fixed by another PR.

Reply:
Happy to help.